tor-spec.txt

```diff
@@ -2169,7 +2169,11 @@ see tor-design.pdf.
 
    Describes the padding capabilities of the relay.
 
-   "1" -- Relay supports circuit-level padding.
+   "1" -- [DEFUNCT] Relay supports circuit-level padding. This version MUST NOT
+          be used as it was also enabled in relays that don't actually support
+          circuit-level padding.
+
+   "2" -- Relay supports the HS circuit setup padding machines (proposal 302)
```

Comment:
Do we want to specifically mention the HS setup padding machine?
Do we want to mention versions?
(1 is 0.4.0.? to 0.4.1.4-rc, 2 is 0.4.1.5-rc and later.)

Reply:
Thanks for the review.

The new plan is to bind protover numbering to specific padding machines as explained by Mike in tor commit msg b14bb12e39d3ccb95a9e5d9cf3eda8428c129520, and hence why I put that there.

I also pushed a fixup a26b772ab that documents the precise versions since it might come useful to someone.